build.gradle.kts

```diff
@@ -66,6 +90,8 @@ dependencies {
     testCompile("junit:junit:4.12")
     testCompile("org.assertj:assertj-core:3.11.0")
     testCompile("com.atlassian.performance.tools:docker-infrastructure:0.1.2")
+    testCompile("com.github.docker-java:docker-java:3.1.1")
+    testCompile("com.atlassian.performance.tools:infrastructure:[4.0.0,5.0.0)")
```

Comment:
This creates a dependency cycle in the `testCompile` configuration, because `infrastructure` depends on `virtual-users`. However, thanks to version ranges, they do not conflict, as long as the `./gradlew currentVersion` fits the range.
We'll have a problem when we'll want to release breaking changes. Then we'll have to deprecate things before breakage (as we should anyway) and release `infrastructure`, which doesn't rely on the deprecated stuff anymore. Then we can force-resolve the conflict via `useVersion` and undo it after we release the major VU and bump it in infra.

In the short-term there's a small problem. When a repo doesn't have the newest release tags, `./gradlew currentVersion` will yield an out-of-date, which can fall out of the range. The solution is to push the tags to the repo. Just FYI.